akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -187,14 +187,14 @@ object ActorSystem {
    * then tries to walk the stack to find the callers class loader, then falls back to the ClassLoader
    * associated with the ActorSystem class.
    *
-   * @see <a href="http://typesafehub.github.io/config/v1.3.1/" target="_blank">The Typesafe Config Library API Documentation</a>
+   * @see [[https://lightbend.github.io/config/v1.3.1/ The Typesafe Config Library API Documentation]]
```

Comment:
Does `[[` work for both javadoc and scaladoc?

Reply:
Hmm yeah I think this should remain normal links

Reply:
Yeah, markdown didn't work for javadoc. I will rollback soon.